app/lib/onlineTasksMapper.js

```diff
@@ -0,0 +1,17 @@
+function map(gpData) {
+  let onlineTasks;
+
+  if (gpData.bookingSystem) {
+    onlineTasks = {};
+    onlineTasks.bookOnlineLink = gpData.bookingSystem.bookOnlineLink;
+  }
+
+  if (gpData.onlineServices && gpData.onlineServices.repeatPrescriptions) {
+    onlineTasks = onlineTasks || {};
+    onlineTasks.repeatPrescriptionOnlineLink = gpData.onlineServices.repeatPrescriptions.url;
+  }
+
+  return onlineTasks;
+}
+
+module.exports = map;
```

Comment:
I feel this would be more readable if it was structured like

return gpHasOnlineTasks(gpData) ? createOnlineTasks(gpData) : undefined;

This would remove the complexity around creating the onlineTasks variable.